### PR TITLE
Add the open source update article

### PR DESCRIPTION
**Pull request by ielerol**
I think this is finally ready to merge. You can preview the article here: http://ielerol.github.io/cfpb.github.io/articles/open-source-update/

there are some broken images in the embedded timeline, but I don't know who put that together or how it could be fixed.

### CODE REVIEW

**Comment by marteki** [comment]
@ascott1 do you have access to the timeline to fix the broken images?

File: _posts/2017-03-27-open source update.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+layout: post
+author: Catherine Farman
+title: "History and future of open source at CFPB"
+permalink: /articles/open-source-update/
+---
+
+
+At CFPB, we develop nearly all of our software publicly on GitHub 
+and release it under a 
+[Public Domain/CC0 license.](https://github.com/cfpb/source-code-policy/blob/gh-pages/TERMS.md)
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: Move the period outside of the link.

File: _posts/2017-03-27-open source update.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+[Public Domain/CC0 license.](https://github.com/cfpb/source-code-policy/blob/gh-pages/TERMS.md) 
+We adopted this policy in order for taxpayers to benefit from the products they pay for, 
+to provide a window into how our agency conducts business, 
+and to increase the quality of our source code through contributions 
+from the wider development community.
+
+Since 2012, we have seen the benefits of an open source by default policy 
+in the following outcomes:
+
+- CFPB’s Github organization lists [182 public repositories](https://github.com/cfpb)
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: *GitHub

File: _posts/2017-03-27-open source update.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+- Projects like eRegulations have been forked and adopted by other government agencies, 
+including the [Bureau of Alcohol, Tobacco, Firearms and Explosives](https://atf-eregs.18f.gov/)
+- We hired a new class of Design and Development fellows in 2015 and 2016, 
+many of whom cited our open source policy and active Github presence
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: *GitHub

File: _posts/2017-03-27-open source update.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+and to increase the quality of our source code through contributions 
+from the wider development community.
+
+Since 2012, we have seen the benefits of an open source by default policy 
+in the following outcomes:
+
+- CFPB’s Github organization lists [182 public repositories](https://github.com/cfpb)
+- Projects like eRegulations have been forked and adopted by other government agencies, 
+including the [Bureau of Alcohol, Tobacco, Firearms and Explosives](https://atf-eregs.18f.gov/)
+- We hired a new class of Design and Development fellows in 2015 and 2016,
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: "Design and Development fellows" ➡️ "Technology and Innovation Fellows"

**Comment by ascott1** [comment]
@marteki the images seem to work ok for me in chrome, but are busted in Safari. Right clicking them opens the correct image... so I'm not sure what's going on. Could/should they be uploaded to a different image host?

File: _posts/2017-03-27-open source update.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+including the [Bureau of Alcohol, Tobacco, Firearms and Explosives](https://atf-eregs.18f.gov/)
+- We hired a new class of Design and Development fellows in 2015 and 2016, 
+many of whom cited our open source policy and active Github presence 
+as a top reason for applying to and joining CFPB.
+- We don’t just code in the open, we now design in the open too, 
+- and [shared our icon library with the Noun Project](https://cfpb.github.io/articles/federal-icons-in-the-open/).
+
+Open source software development has been critical at CFPB since we opened our doors. 
+In April 2012, just nine months after the Bureau opened, 
+we released our [Open Source Policy](http://www.consumerfinance.gov/blog/the-cfpbs-source-code-policy-open-and-shared/),
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: "Open Source Policy" ➡️ "Source Code Policy"

File: _posts/2017-03-27-open source update.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+in [2013](http://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking/), 
+[2014](http://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking-again/),  
+and [2015](http://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking-2015/), 
+working with developers to create data visualizations using the [Consumer Complaint Database](http://www.consumerfinance.gov/complaintdatabase/) 
+and [Home Mortgage Disclosure Act](http://www.consumerfinance.gov/hmda) data.
+- Software developers from 18F’s eRegs team attended the General Services Administration’s 
+[GovTechHack](https://18f.gsa.gov/2015/04/02/govtechhack-hacking-for-civic-improvement/) in 2015, 
+where they focused on improving the software’s local setup process. 
+They succeeded in decreasing the setup time for eRegs from 2 hours to 15 minutes.
+- CFPB Developer Catherine Farman gathered a team of designers and coders
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: *developer

File: _posts/2017-03-27-open source update.md (hunk 5 above)
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: *https here and every other consumerfinance.gov URL in the piece.

File: _posts/2017-03-27-open source update.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+- Ensuring our project setup documentation is up to date by holding regular 
+[README Refresh Days](https://cfpb.github.io/articles/readme-refresh-day/).
+- Sharing our experiences with our fellow federal agencies and learning from them too, 
+so we can incorporate their efforts into our own process.
+- [Educating developers](https://github.com/cfpb/data-viz-workshop) about CFPB’s open source projects and data sets, 
+inspired by [18F’s data visualization workshop](https://openglobe.github.io/cfpb-dataviz/).
+- Increasing our community outreach by presenting our work at conferences and meetups.
+
+In March, the [White House released a draft federal open source software policy for public comment](https://www.whitehouse.gov/blog/2016/03/09/leveraging-american-ingenuity-through-reusable-and-open-source-software). 
+Members of the public gave feedback and suggested changes via Github.com.
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: *GitHub

File: _posts/2017-03-27-open source update.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+[README Refresh Days](https://cfpb.github.io/articles/readme-refresh-day/).
+- Sharing our experiences with our fellow federal agencies and learning from them too, 
+so we can incorporate their efforts into our own process.
+- [Educating developers](https://github.com/cfpb/data-viz-workshop) about CFPB’s open source projects and data sets, 
+inspired by [18F’s data visualization workshop](https://openglobe.github.io/cfpb-dataviz/).
+- Increasing our community outreach by presenting our work at conferences and meetups.
+
+In March, the [White House released a draft federal open source software policy for public comment](https://www.whitehouse.gov/blog/2016/03/09/leveraging-american-ingenuity-through-reusable-and-open-source-software). 
+Members of the public gave feedback and suggested changes via Github.com. 
+To date the public has contributed 18
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: - Insert comma after "To date"
- *20

File: _posts/2017-03-27-open source update.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+To date the public has contributed 18 
+[merged pull requests](https://github.com/WhiteHouse/source-code-policy/pulls?q=is%3Apr+is%3Aclosed) 
+and over 200 [discussion threads](https://github.com/whitehouse/source-code-policy/issues/).
+
+If you have thoughts about how we can or other federal agencies 
+might continue to evolve our approach to open source software, 
+we’d encourage you to 
+[discuss the Federal Open Source Policy on GitHub.](https://github.com/whitehouse/source-code-policy/issues/)
+
+<iframe width="640" height="650" src="https://cdn.knightlab.com/libs/timeline3/latest/embed/index.html?source=1Hr66qNKhDf85BRFeyBzTQ9rN5Z84B_JP-_ZSqH2YIIw&font=Default&lang=en&initial_zoom=2&height=650"> </iframe>
```
Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: Remove space inside `<iframe>` tags.

**Comment by marteki** [comment]
As for the timeline: we decided in an editorial meeting about this to remove the timeline from the article, in the interest of publishing this in a relatively timely manner.

(Based on our quick investigation, the image issue seems to be related to HTTPS images.)

File: _posts/2017-03-27-open source update.md (hunk 6 above)
Comment (ielerol) [inline _posts/2017-03-27-open source update.md]: you're asking here for the word "coders" on line 68 to change to "developers," right?

Comment (Scotchester) [inline _posts/2017-03-27-open source update.md]: Yes. Sorry, that could have been more clear and grammatically correct.

**Comment by Scotchester** [comment]
Superseded by #113

**Pull request closed without merging**